test/integration/default/controls/services_spec.rb

```diff
@@ -1,3 +1,7 @@
+# Temporary `if` due to `opensuse-leap-15` bug re: `service`
+if os[:name] == 'suse'
+  puts "[Skip `service`-based tests due to `opensuse-leap-15` bug]"
```

Comment:
Perhaps adding a reference to the bug in the comment, lile

```
Skip `service`-based tests due to `opensuse-leap-15` detection bug (see https://github.com/inspec/train/issues/377)
```
?

Reply:
Good idea, done.